### PR TITLE
Email State Admin from State Dashboard when Affiliation is Pending

### LINKED ISSUE #2771: Allow users to email state admin from all relevant dashboards
Users should be able to email state admin from the pending dashboard using a mailto: link that pulls from the database the holds the state admins we've approved. 

For release, we'll only allow one state admin per state therefore one state admin to pull into this link. 

### This task is done when...
 - [ ] a user is able to email their state admin from their dashboard
 - [ ] Displays approved, denied, and revoked

### PR DESCRIPTION
**Pull request by radavis**
Resolves #2771

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [x] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated
- [x] Changelog is updated as appropriate

### This feature is done when...

- [x] QA has approved the experience
- [x] Design has approved the experience
- [ ] Product has approved the experience

### Steps to verify this PR:

* Create an Okta User: https://test.reg.idm.cms.gov/registration.html?appid=eapd
* Login
* Select 'Alaska'
* Verify that 'State Administrator' link is present with `href='mailto:...'`
* Login into the em@il user account and verify the new user(s) requested for Alaska is in pending deliveries.

Repeat steps above with another user, a different state (e.g. - 'Alabama') and verify 'State Administrator' is present with 'href=mailto:CMS-EAPD@cms.hhs.gov' (no state admin assigned, yet).

### CODE REVIEW

File: bin/preview-deploy/aws.sh [hunk 1]
```diff
@@ -116,6 +116,7 @@ function createNewInstance() {
     --subnet-id "$AWS_SUBNET" \
     --tag-specification "ResourceType=instance,Tags=[{Key=Name,Value=eAPD PR $PR_NUM},{Key=environment,Value=preview},{Key=github-pr,Value=${PR_NUM}}]" \
     --user-data file://aws.user-data.sh \
+    --key-name radavis \
```
Comment (thetif) [inline bin/preview-deploy/aws.sh:119]: are you still testing this?

**Review by thetif — APPROVED** [review]

File: bin/preview-deploy/aws.sh (hunk 1 above)
Comment (radavis) [reply bin/preview-deploy/aws.sh]: Resolved

**Review by tbolt — APPROVED** [review]
Looks good

**Review by beparticular — CHANGES_REQUESTED** [review]
Apologies again for not catching this sooner - can the text be changed to: 

Contact the State Administrator for more information.

**Review by beparticular — APPROVED** [review]

**Review by eStar386 — COMMENTED** [review]
There is an accessibility and bug that prevents the screen from redirecting the user to the login screen this is fixed with a larger pr #2891

**Comment by eStar386** [comment]
This functionality has accessibility issues (wave) and a bug that prohibits it from redirecting to the pending deliveries screen and is within #2838
Test Script is located at [2771 ](https://drive.google.com/file/d/1HFm8OB6jDnFjKF4jvbbUIXcz-56P64Wn/view?usp=sharing)

**Comment by eStar386** [comment]
During backend validation, an issue was discovered with the database saving user information. Update the test steps above to assist with the investigation.

**Comment by eStar386** [comment]
@radavis - This restates the bug more eloquently. It mirrors the above comment.
Type of Bug Title: Backend Validation Failure 

Role Affected: em@il.com

Description of issue: During backend validation, an issue was discovered with the database saving new user pending deliveries. The new user state affiliation must be sent to the em@il user account whenever the default case of Alaska is used. 

Expected Result:  New user added with a state affiliation of Alaska.  The New user requests received and appears on the state admin dashboard. 

Actual Result: The new user has a state affiliation of Alaska with the pending deliveries request. However, the em@il user doesn’t have the request in the dashboard.


Steps to Reproduce 

•	Create an Okta User: https://test.reg.idm.cms.gov/registration.html?appid=eapd
•	Login
•	Select 'Alaska'
•	Verify that 'State Administrator' link is present with href='mailto:...'
•	Login into the em@il.com account and verify the new user(s) requested for Alaska is in pending deliveries.

**Comment by radavis** [comment]
@eStar386 I was able to confirm this issue.

Requests from the front-end when viewing the `/state-admin` path were returning `502 Bad Gateway` responses. Thus, no information was being retrieved from the back-end.

Merging changes from master seem to have corrected this problem. Please re-test when you have the time. Thank you.

**Comment by eStar386** [comment]
Retested and this resolves the bug.

**Review by eStar386 — APPROVED** [review]

**Review by jeromeleecms — COMMENTED** [review]
The scope of the PR does not reflect the scope of the actual issue. We should also have this same functionality for the "Denied Dashboard" as well:

image

Currently the text does not link to anything. If we do change the scope of this issue, let's change the title to include all dashboards -- or I'm OK with switching it to a separate issue to resolve. @thetif

**Review by thetif — CHANGES_REQUESTED** [review]

File: bin/preview-deploy/aws.sh [hunk 2]
```diff
@@ -116,7 +116,7 @@ function createNewInstance() {
     --subnet-id "$AWS_SUBNET" \
     --tag-specification "ResourceType=instance,Tags=[{Key=Name,Value=eAPD PR $PR_NUM},{Key=environment,Value=preview},{Key=github-pr,Value=${PR_NUM}}]" \
     --user-data file://aws.user-data.sh \
-    --key-name eapd_bbrooks \
+    --key-name radavis \
     | jq -r -c '.Instances[0].InstanceId'
```
[the remark this answers is not in the document]
Comment (knollfear) [reply bin/preview-deploy/aws.sh:120]: sure.  I have no idea why this is part of the MR, I didn't look at this part.

**Review by tbolt — APPROVED** [review]
Nice work

**Review by thetif — APPROVED** [review]

**Review by NAretakis — APPROVED** [review]
I see the admin email, clicking it works too!! Good to go!

**Review by eStar386 — APPROVED** [review]
Passed - the denied and revoked panels are added. Testing information above.

**Pull request merged**